hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/StorageImpl.java

```
@@ -66,6 +67,11 @@ public void clear(boolean isDuringShutdown) {
     }
 
     @Override
+    public Iterator<R> iterator(boolean failFast) {
```

if internal impl supports fail fast, it should fail. I'm using it on ee part.